content/wiki/cucumber-backgrounder.md

```diff
@@ -565,7 +565,7 @@ When /"Bob" logs in/ do |user|
 end
 ```
 
-In fact, this Step Definition should be further refactored into a method. The  method can reside in the same `.rb` file as the Step Definition. This both simplifies the step, and encourages the reuse of the resulting method.
+In fact, this Step Definition should be further refactored into a method. The method can reside in the same `.rb` file as the Step Definition. This both simplifies the step, and encourages the reuse of the resulting method.
 
 It also adds immeasurably to easy comprehension for people unfamiliar with the project history who may later have cause to review your tests. Can you spell **M-A-I-N-T-E-N-A-N-C-E**?
```

Comment:
> Can you spell **M-A-I-N-T-E-N-A-N-C-E**?

I know the intention is good here, but it can easily come across as snarky.

Reply:
Oh, I agree. Didn't look into the rest of the text. Let's update this

Reply:
Updated this sentence. The whole doc still needs to be reviewed/generalized (as mentioned in the TODO list).